astropy/io/ascii/tests/test_cds.py

```diff
@@ -558,3 +558,70 @@ def test_write_qtable():
     result = out.getvalue()
     assert "Description of a" in result
     assert "Description of b" in result
+def generate_mrt_file(filename="test.mrt"):
+    """Creates an MRT file with NaN and constant columns."""
+    nan_column = Column(
+        np.full((11,), np.nan), name="nans", unit="m", dtype=float
```

Comment:
```suggestion
        np.full((3,), np.nan), name="nans", unit="m", dtype=float
```
I still think it's much easier and less redundant code to just use the existing `test_dat` with a modified column, but if you do use your own, 2 rows would be enough to prove the case, and 3 are plenty.